### PR TITLE
MOD-7499 Add JSON ACL category

### PR DESCRIPTION
**Pull request by ephraimfeldblum**
[MOD-7499](https://redislabs.atlassian.net/browse/MOD-7499)

[MOD-7499]: https://redislabs.atlassian.net/browse/MOD-7499?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -82,7 +82,7 @@ commands:
     parameters:
       redis_version:
         type: string
-        default: "7.2"
```
Comment (eranhd) [inline .circleci/config.yml:85]: why do we need to change it in circleci?

Comment (iddm) [reply .circleci/config.yml:85]: I think we should delete it, I agree. As a separate commit should be fine.

File: .github/actions/build-json-module-and-redis-with-cargo/action.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ inputs:
   redis-ref:
     description: 'Redis version to build'
     required: true
-    default: '7.2.1'
```
Comment (iddm) [inline .github/actions/build-json-module-and-redis-with-cargo/action.yml:9]: Excuse me, do we really need to be that precise to specify `7.4.1` and not just `7.4`?

File: redis_json/Cargo.toml [hunk 3]
```diff
@@ -22,8 +22,8 @@ ijson.workspace = true
 serde_json.workspace = true
 serde.workspace = true
 libc = "0.2"
-redis-module ={ git="https://github.com/RedisLabsModules/redismodule-rs", tag="v2.0.8", default-features = false, features = ["min-redis-compatibility-version-7-2"] }
-redis-module-macros = { git="https://github.com/RedisLabsModules/redismodule-rs", tag="v2.0.8" }
```
Comment (iddm) [inline redis_json/Cargo.toml:26]: We should release the `redismodule-rs` version and use the new tag here.

**Review by iddm — CHANGES_REQUESTED** [review]
We need to release the redismodule-rs version and depend on it here, instead of depending on the master branch.

File: .github/actions/build-json-module-and-redis-with-cargo/action.yml (hunk 2 above)
Comment (ephraimfeldblum) [reply .github/actions/build-json-module-and-redis-with-cargo/action.yml:9]: these were the result of a `sed` from 7.2 to 7.4 :shrug:

**Review by iddm — APPROVED** [review]

**Comment by eranhd** [comment]
Please update the description to include detailed information.

**Review by eranhd — APPROVED** [review]

**Pull request merged**